src/OnboardingContext.tsx

```diff
@@ -36,6 +36,7 @@ export const OnboardingProvider: FunctionComponent<OnboardingProviderProps> = ({
   const completeOnboarding = () => {
     StorageUtils.setIsOnboardingComplete()
     setIsOnboardingComplete(true)
+    return Promise.resolve()
```

Comment:
What do you think about using async await here? Not sure if this will guarantee that StorageUtils.setIsOnboardingComplete() will finish before the chained then will resolve with this set up, if thats the intent here.

Reply:
Makes sense!